lib/rules/rules.json

```diff
@@ -47,8 +47,8 @@
         "ES6": true
     },
     "first": {
-        "compatible": true,
-        "alternative": "Array.prototype.slice() or arr[0]"
+      "compatible": true,
+      "alternative": "Array.prototype.slice()"
```

Comment:
Why did you remove `or arr[0]` part?

Reply:
Please run `test:unit` and you'll see this error :)